src/assets/mapstorage/readme.md

```diff
@@ -0,0 +1 @@
+Adding mapstorage folder
```

Comment:
I wouldn't put these README.md files here unless there is content to put in them. It makes it harder to understand the purpose of the folder.